<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpLoad</class>
 <widget class="QDialog" name="UpLoad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">image: url(:/pick_upload/background.jpg);</string>
  </property>
  <widget class="QProgressBar" name="clientProgressBar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="clientStatusLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>状态：等待打开文件！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>561</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
</string>
   </property>
   <property name="text">
    <string>如需开店，上传物品名称，图片，价格输入在txt文档打包后发送。工作人员将在后台审核后为您服务</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
